banks-interface/src/lib.rs

```diff
@@ -44,6 +57,10 @@ pub trait Banks {
         -> Option<TransactionStatus>;
     async fn get_slot_with_context(commitment: CommitmentLevel) -> Slot;
     async fn get_block_height_with_context(commitment: CommitmentLevel) -> u64;
+    async fn process_transaction_with_preflight(
```

Comment:
For consistency, should this be `process_transaction_with_preflight_and_context`?  Or have I misunderstood what should be in the trait part?

Reply:
I was trying to avoid having to do `process_transaction_with_preflight_and_commitment_and_context`. But I see your point about calling out the tacit `context, and it seems like we're not actually consistent wrt commitment... so can get away with `process_transaction_with_preflight_and_context`

Reply:
I came back around to just using the verbose method name.